lib/customers.js

```diff
@@ -332,4 +338,40 @@ function batch () {
     }, customers)))
 }
 
-module.exports = {add, get, batch, getById, update}
+function updatePhotoCard (id, data) {
+  let imageData = _.get('idCardPhotoData', data)
+
+  if (_.isEmpty(imageData)) {
+    return Promise.resolve(data)
+  }
+
+  data = _.omit('idCardPhotoData', data)
+
+  // decode the binary image
+  imageData = Buffer.from(imageData, 'base64')
```

Comment:
Need to check the performance of this. It's the simplest way to do it, which is good, but for huge images it might be a lot faster and use less bandwidth to send the image in binary.